twisted/conch/scripts/ckeygen.py

```diff
@@ -33,10 +33,10 @@ class GeneralOptions(usage.Options):
                      ['type', 't', None, 'Specify type of key to create.'],
                      ['comment', 'C', None, 'Provide new comment.'],
                      ['newpass', 'N', None, 'Provide new passphrase.'],
-                     ['pass', 'P', None, 'Provide old passphrase.']]
+                     ['pass', 'P', None, 'Provide old passphrase.'],
+                     ['fingerprint', 'l', 'sha256-base64', 'Fingerprint format of key file.']]
 
-    optFlags = [['fingerprint', 'l', 'Show fingerprint of key file.'],
-                ['changepass', 'p', 'Change passphrase of private key file.'],
+    optFlags = [['changepass', 'p', 'Change passphrase of private key file.'],
```

Comment:
The letter? No.  but hopefully we can stay in the spirit.  It seems like this meets it either way though; you're just adding stuff?  You're always allowed to add stuff.